### ISSUE TITLE
Add an "Inapplicable" option to the accessibility statement

### ISSUE THREAD

**Issue by WilcoFiers** [w3c/wai-evaluation-tools-list#78]
On the accessibility statement filter, it seems this filter is going to encourage organisations to create accessibility statement for tools that really don't need one, just so they don't get filtered out the list. APIs for example do not have a user interface. There is nothing to declare in an accessibility statement about them.

## Suggestion

1. To the submission form, add an "inapplicable" toggle to the accessibility statement question. This should disable the accessibility statement form control

2. On tools list itself, ensure that tools that indicated accessibility statements are inapplicable are not removed by the "Accessibility statement" filter. Show "Accessibility statement: Inapplicable" as under "Show more details"  for that tool.

**Comment by bakkenb** [comment]
I agree with the rationale stated by Wilco. I am not a developer so I am assuming the logic that some tools don't need a statement is correct. 

Only thing I would change is to change the name of the toggle to "not applicable."

**Comment by WilcoFiers** [comment]
"Not applicable" makes sense. +1

**Comment by EricVelleman** [comment]
This would mean that if someone uses this filter, they would still also get results without an Accessibility Statement. Not sure how this helps users of the page to make a selection?

**Comment by VeraLange** [comment]
I would suggest to then add "Not applicable"  where the accessibility statement is displayed in the tool itself.

That would result in users getting tools that:

- Have provided a statement
- Or: that do not require one

And filtering out those that don't have one helps the user somewhat.

_However, this might result in tool vendors simply choosing "Not applicable", even in cases where they should provide a statement._

TL;DR: not sure how to solve this one.

**Comment by WilcoFiers** [comment]
Tool submissions are reviewed before they are posted. If a tool with a user interface sets this to "not applicable" the reviewer should probably have a conversation with the tool vendor, just like they should for other obviously incorrect submissions.

**Comment by mhansma96** [comment]
Vera's solution has been implemented, should be reviewed by EO
